src/instructlab/configuration.py

```diff
@@ -657,6 +660,7 @@ def get_default_config() -> Config:
             num_epochs=10,
             effective_batch_size=3840,
             save_samples=250000,
+            checkpoint_at_epoch=True,
```

Comment:
nit: I suspect this is not needed since it's already set to True in `fill_defaults`

Reply:
Yeah likely true

Reply:
I moved the default into the pydantic model- that seems cleaner than introducing a potential `fill_defaults` mismatch

Reply:
FYI I am going to rip these `fill_defaults` off post 0.18: https://github.com/instructlab/instructlab/pull/1985